api-reference/10 UI Components/dxCalendar/1 Configuration/value.md

```diff
@@ -5,26 +5,26 @@ default: null
 ---
 ---
 ##### shortDescription
-An object or a value specifying the date and time currently selected in the calendar.
+An object or a value that specifies the date and time currently selected in the calendar. Available only for *'single'* [selection mode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
 
 ---
-You can specify the current UI component value using any of the following formats.
+You can use the following formats to specify the current UI component value:
```

Comment:
"the current UI component value" - seems like a complicated phrase. I'm not sure if there's any specific reason you want to keep these words. But if you don't mind rewriting, I'd say something simple like this:

```suggestion
You can use the following date formats:
```